src/system/ZAuthModule/Resources/views/Authentication/EmailLoginBlock.topnav.html.twig

```diff
@@ -1,5 +1,5 @@
 {% form_theme form with [
-'ZikulaFormExtensionBundle:Form:bootstrap_3_zikula_admin_layout.html.twig',
+'bootstrap_3_layout.html.twig',
```

Comment:
that part of the change reverted here: https://github.com/zikula/core/commit/8e97308aa9928e1b2a96a1a42b613c4f5b701bd8